specs/_features/eip7594/das-core.md

```diff
@@ -221,6 +224,40 @@ def get_data_column_sidecars(signed_block: SignedBeaconBlock,
     return sidecars
 ```
 
+#### `get_extended_sample_count`
+
+```python
+def get_extended_sample_count(allowed_failures: uint64) -> uint64:
+    assert 0 <= allowed_failures <= NUMBER_OF_COLUMNS // 2
+
+    def math_comb(n: int, k: int) -> int:
```

Comment:
Should we use a library instead of implementing our own?
```python
>>> from math import comb
>>> comb(10, 3)
120
```
https://stackoverflow.com/questions/4941753/is-there-a-math-ncr-function-in-python

Reply:
This was discussed in https://github.com/ethereum/consensus-specs/pull/3782#discussion_r1619881097 and in comments after that.

In short:
- imports are not really used in spec code
- hypergeom_cdf implements hypergeom.cdf
- math_comb implements math.comb
- math.comb is also quite new (Python 3.9+) and fails on negative values (strange choice from the Python library version).
This is only for the spec. Client implementations can of course use respective library functions.